src/runtime/rpc/rpc_endpoint.h

```diff
@@ -70,6 +70,12 @@ class RPCEndpoint {
  public:
   /*! \brief virtual destructor */
   ~RPCEndpoint();
+
+  /*!
+   *  \brief The server shutdown cleanup function.
+   */
+  void Shutdown();
```

Comment:
We should have a more descriptive docstring here, and in the destructor.  For the destructor, we should say that it closes the connection, if the connection hasn't already been closed.  For the Shutdown function, we should 

* Shutdown has no effect if the connection has already been shut down.
* Shutdown will wait for all output currently queued from the RPC connection (i.e. The user doesn't need to wait for completion before calling Shutdown.)
* Any further use of objects that depended on the endpoint (e.g. A `tvm.nd.array` allocated on the remote RPC session) may throw an exception when used.